api/v1alpha1/inferencepool_types.go

```diff
@@ -40,35 +40,35 @@ type LLMServerPoolSpec struct {
 	TargetPort int32 `json:"targetPort,omitempty"`
```

Comment:
This should match [the target port on a k8s Service which supports an int or string](https://github.com/kubernetes/kubernetes/blob/7a504aa97bf3006848b1dc9281aa039ad180ae81/pkg/apis/core/types.go#L4869).

Reply:
Just for posterity: https://github.com/kubernetes-sigs/llm-instance-gateway/issues/98 this issue was opened. So we are going to consider figuring out these details out of scope for the current PR